### DISCUSSION TITLE
integrate a Tor WebTunnel bridge on Nextcloud AIO?

### DISCUSSION THREAD

**Discussion by Gregg3** [nextcloud/all-in-one#5703] [Ideas] [answered]
First of all, thank you for Nextcloud All-In-One. It is amazingly simple to install on my vps and keep it updated.

I haven't been able to find a Nextcloud add-in for running a Tor bridge. Am I just missing it?
Or maybe someone could point me to instructions on getting both Nextcloud AIO and a WebTunnel bridge working on the same host?

for reference as to why I'm interested, please see this blog post on [torproject.org](https://blog.torproject.org/call-for-webtunnel-bridges/)

**Comment by szaimen** [comment, marked as answer]
Hi, interesting idea! [The community container feature](https://github.com/nextcloud/all-in-one/tree/main/community-containers#community-containers) could be used for the [docker container of torproject](https://blog.torproject.org/call-for-webtunnel-bridges/#:~:text=how%20to%20run%20a%20tor%20webtunnel%20bridge). See [this doc](https://github.com/nextcloud/all-in-one/tree/main/community-containers#how-to-add-containers) how to add a container to the list. Let me know if you need any help with this!
